### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]: :Jaws and NVDA are not announcing the action performed when "Alt+B" shortcut key is used in "Run on web app" dialog

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1463]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Create a new java project.
3. Tab to and select Project ->Azure->Run on web app
4. "Run on  web app" dialog will open. 
5. In "Run on web app" dialog , tab to  and select "Use Existing web app" radio buttons.
6. Use "Alt +B" to expand/collapse the Before launch expand/collapse button
7. Observe the behaviour of Jaws and NVDA 

**Actual Result:**
Jaws and NVDA are not announcing the performed action when Alt+B shortcut key is used

**Expected Result:**
Jaws and NVDA should announce the performed action when short cut key is used
They should announce as
Before Launch :build Artifact's ,Activate tool window expanded Alt+B

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![a4b9711b](https://user-images.githubusercontent.com/22956130/36101797-5e0c26cc-1030-11e8-925a-7e69a0bbb5d1.jpg)
![run on web app-expand-not annoucning](https://user-images.githubusercontent.com/22956130/36101822-6f031f08-1030-11e8-8e55-62240e98fe75.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749929
